source/pug/templates/base-for-django.pug

```diff
@@ -0,0 +1,18 @@
+//-
+  This template is used to generate HTML for usage in Mezzanine.
+  There are {{ TOKENS }} in place for Mezzanine to target with CMS data.
+  `npm build` will compile this to `/dest/opportunity/template/index.html`
+
+extends ../templates/master.pug
+
+block masterParams
+  - let isMezzanine = true
+  - let pageTitle = `{{ page.meta_title }}`;
+
+include ../partials/sign-up-box.pug
```

Comment:
What's this doing?

Reply:
Ah, I think actually nothing. That was an artifact from the previous opportunity template, but it's an unused mixin apparently.